accounting/accounting_cli_functions.py

```diff
@@ -168,76 +168,52 @@ def add_job_records(dataframe):
     return job_records
 
 
-def view_jobs_run_by_username(conn, username, output_file):
-    # look up userid with username
-    userid = get_uid(username)
-    # get the information pertaining to a user in the jobs DB
-    select_stmt = (
-        "SELECT userid,id,t_submit,t_run,t_inactive,ranks,R FROM jobs where userid=?"
-    )
-    dataframe = pd.read_sql_query(select_stmt, conn, params=(userid,))
-    # if the length of dataframe is 0, that means
-    # the user specified was not found in the table
-    if len(dataframe.index) == 0:
-        return "User not found in jobs table"
-    else:
-        job_records = add_job_records(dataframe)
-        if output_file is None:
-            print_job_records(job_records)
+def view_job_records(conn, output_file, args):
+    # find out which args were passed and place them in a dict
+    params = {}
+    params_list = []
+
+    for key, val in vars(args).items():
+        if val != None and key != "path" and key != "output_file" and key != "func":
```

Comment:
if you add a new general option to the tool, you'll have to update this loop check, which people will likely forget to do and could lead to errors.  Instead, how about only matching options this function cares about.

```
params_i_care_about = ("user", "after_start_time", "before_start_time", "jobdid)
for key, val in vars(args).items(): 
    if val != None and key in params_i_care_about:
       ...
```